sup3r/preprocessing/rasterizers/extended.py

```diff
@@ -193,7 +193,8 @@ def get_lat_lon(self):
         return self._get_flat_data_lat_lon()
 
     def _get_flat_data_lat_lon(self):
-        """Get lat lon for flattened source data."""
+        """Get lat lon for flattened source data. Output is shape (y, x, 2)
+        where 2 is (lat, lon)"""
         if hasattr(self.full_lat_lon, 'vindex'):
             return self.full_lat_lon.vindex[self.raster_index]
-        return self.full_lat_lon[self.raster_index.flatten]
+        return self.full_lat_lon[self.raster_index]
```

Comment:
Good catch here. This getting through must mean there was no test coverage for the h5 rasterizers with data loaded in memory? What did you run to hit this? Would you mind adding something similar as a simple smoke test, in `test_rasterizer_general.py`?

Reply:
numpy or dask? With `chunks=None` it should be a numpy array and then run into the misplaced `flatten`

Reply:
oops sorry, edited the comment, lat/lon is always a dask array even with chunks=None. Here's the draft test (stupid simple)
```
def test_preloaded_h5():
    """test preload of h5 file"""
    rasterizer = Rasterizer(
        file_paths=pytest.FP_WTK, target=(39.01, -105.15), shape=(20, 20),
        chunks=None
    )

Reply:
It's possible that i ran into this error before https://github.com/NREL/sup3r/pull/247 got merged and then rebased and now this code is never used?

Reply:
Yeah this is weird actually bc ``Sup3rX.compute`` only loads ``data_vars`` into memory: https://github.com/NREL/sup3r/blob/94eebae081cb07f4dd12132b6ec2a10b73a8dfc6/sup3r/preprocessing/accessor.py#L233 
and ``chunks=None`` forces a call to ``compute`` after rasterization is complete anyway: https://github.com/NREL/sup3r/blob/94eebae081cb07f4dd12132b6ec2a10b73a8dfc6/sup3r/preprocessing/rasterizers/extended.py#L113. 
It seems like it would take some funky manual method calls to get to ``_get_flat_data_lat_lon`` with a ``numpy`` array at the current version but I don't remember changing anything in that PR that would have impacted this. Either way, I added the test and coordinate compute in PR #251.

Reply:
Yeah definitely an edge case but i ran into it at some point and we should keep the fix for posterity